eng/pipelines/docs-to-blob.yml

```diff
@@ -0,0 +1,40 @@
+#variables => $(TargetPipelineProjectId), $(TargetPipelineDefinition), $(TargetPipelineBranch), $(azure-sdk-doc-sas-key) (from keyvault)
+jobs:
+  - job: CopyDocsToBlob
+    variables:
+      - template: templates/variables/globals.yml
+    pool:
+      vmImage: windows-2019
+    steps:
+      - task: DownloadPipelineArtifact@2
+        displayName: "Download Pipeline Artifact"
+        inputs:
+          buildType: specific
+          project: $(TargetPipelineProjectId) #590cfd2a-581c-4dcb-a12e-6568ce786175
+          definition: $(TargetPipelineDefinition) #54
+          buildVersionToDownload: latestFromBranch
+          branchName: $(TargetPipelineBranch) #refs/pull/5252/merge
+
+      - pwsh: ls
+        workingDirectory: $(Pipeline.Workspace)
+        displayName: View Downloaded Artifacts
+
+      - pwsh: |
```

Comment:
You can reference the @azure-sdk-tools like I do [here.](https://github.com/Azure/azure-sdk-for-python/pull/7854/files?file-filters%5B%5D=.rst&file-filters%5B%5D=.yml#diff-16bb2670d4a97c4059cbfb6799b6d69a)

Reply:
Actually, we need to discuss about this yaml, for testing I have been using Dozie's yaml and pipeline set up in playground. Do we want to centralize the yaml for this and just set different pipelines with respective args?

Reply:
Deleted in favor of the yaml under `azure-sdk-build-tools` repo on devops